alloc/src/chain.rs

```diff
@@ -238,10 +238,7 @@ unsafe impl<A: Allocator + Clone> Allocator for ChainAllocator<A> {
 
         debug_assert!(chain_node.remaining_capacity() >= layout.size());
 
-        let result = chain_node.linear.allocate(layout);
-        // If this fails, there's a bug in the allocator.
-        debug_assert!(result.is_ok());
-        result
```

Comment:
Was there a specific reason to get rid of this?

Reply:
It was causing the fuzzer to spuriously fail.  Do you know of a way to control whether this assert appears using `cfg`?

Reply:
found it